lib/ExpensiMark.js

```diff
@@ -346,6 +348,40 @@ export default class ExpensiMark {
         return replacedText;
     }
 
+    /**
+     * Map the block to newline
+     *
+     * @param {String} htmlString
+     *
+     * @returns {String}
+     */
+    mapBlock(htmlString) {
+        const splitText = htmlString.split('[block]');
+        let joinedText = '';
+
+        // delete empty spaces at the end
+        while (splitText.length) {
+            if (splitText[splitText.length - 1].trim().length === 0 && !splitText[splitText.length - 1].match(/\n/)) {
+                splitText.pop();
+            } else {
+                break;
```

Comment:
break early instead

```
if (!condition) {
    break;
}
splitText.pop()
```

Reply:
yes, it's better. we do not have here these lint rules (early return)  I will suggest these lint rules be added to this repo also.